### PR TITLE
Fix OCP install links

### PR DESCRIPTION
**Pull request by dhaiducek**

### CODE REVIEW

File: clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ Cluster lifecycle requires two types of clusters: _hub clusters_ and _managed cl
 
 The hub cluster is the {ocp-short} (or {acm-short}) main cluster with the {mce-short} automatically installed. You can create, manage, and monitor other Kubernetes clusters with the hub cluster. You can create clusters by using the hub cluster, while you can also import existing clusters to be managed by the hub cluster.
 
-When you create a managed cluster, the cluster is created using the {ocp} cluster installer with the Hive resource. You can find more information about the process of installing clusters with the {ocp-short} installer by reading link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html-single/installing/index[Installing and configuring {ocp-short} clusters] in the {ocp-short} documentation.  
+When you create a managed cluster, the cluster is created using the {ocp} cluster installer with the Hive resource. You can find more information about the process of installing clusters with the {ocp-short} installer by reading link:https://docs.redhat.com/en/documentation/openshift_container_platform_installation/4.15[Installing and configuring {ocp-short} clusters] in the {ocp-short} documentation.
```
Comment (dhaiducek) [inline clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc:8]: These `openshift_container_platform_installation` links are the links that are inadvisable and potentially subject to change, but there isn't any other installation landing page. However, these links are also distinctive, and there aren't many of them, so should be easy to change if/when the URLs do get updated.

**Review by swopebe — APPROVED** [review]

**Comment by dockerymick** [comment]
Since this is approved, merging the PR

**Pull request merged**